src/AppEntry.tsx

```diff
@@ -59,12 +60,20 @@ const AppWithKeycloak: React.FunctionComponent = () => {
   );
 };
 
+const AppWithConfig: React.FunctionComponent = () => {
```

Comment:
@pmuir I created an intermediate element that will display the AppServicesLoading while the config is loading